src/Pact/Types/SizeOf.hs

```diff
@@ -134,3 +147,95 @@ instance SizeOf UTCTime where
   -- Internally 'UTCTime' is just a 64-bit count of 'microseconds'
   sizeOf ti =
     (constructorCost 1) + (sizeOf (toPosixTimestampMicros ti))
+
+instance SizeOf Bool where
+  sizeOf _ = 0
```

Comment:
```suggestion
  sizeOf _ = 1
```

Reply:
unless this is that enum constructor thing? if not, this change also applies to `DefType` etc too

Reply:
I'll change it to 1 word.